mathesar/serializers.py

```diff
@@ -81,6 +81,11 @@ def to_representation(self, instance):
         return instance._asdict()
 
 
+class TablePreviewSerializer(serializers.Serializer):
+    name = serializers.CharField()
```

Comment:
I don't think it makes sense to require name on input, we already know which table this is. I recommend making this `required=False`.

Reply:
What would you think of leaving it out entirely?  I don't think it's really needed at all.

Reply:
I think that's fine.